### PR TITLE
gluon-status-page: add domain and site to status page

### PR DESCRIPTION
**Pull request by lemoer**
Partly implements #1415

### CODE REVIEW

**Comment by rotanid** [comment]
@lemoer thanks. could you please look at #1402 and the discussion there and maybe also take "role" into your PR as in 1402?

File: package/gluon-status-page/files/lib/gluon/status-page/view/status-page.html [hunk 1]
```diff
@@ -116,6 +116,8 @@ <h2><%:Overview%></h2>
 					<% if nodeinfo.software.fastd then -%>
 						<dt><%:Mesh VPN%></dt><dd><%| enabled(nodeinfo.software.fastd.enabled) %></dd>
 					<%- end %>
+					<dt><%:Domain%></dt><dd><%| site.domain_names[nodeinfo.system.domain_code]() %></dd>
```
Comment (mweinelt) [inline package/gluon-status-page/files/lib/gluon/status-page/view/status-page.html]: let's assume the firmware isn't multidomain, what happens then?

**Comment by CodeFetch** [comment]
@lemoer Nice... Can you please try whether the design is still responsive? I did a similar change a while ago which lead to "jumps" in the viewports. I did a fixup of the CSS which I totally forgot about. If you experience similar problems I can have a look next week.

**Comment by mweinelt** [comment]
Live with multidomain firmware here: http://[2001:67c:2ed8:1001:928d:78ff:fe23:af1c]/cgi-bin/status

I think site should appear before domain, other than that it lgtm.

**Comment by rotanid** [comment]
i can confirm it works for a non-multidomain site.
note: i tested the commit before the swap of the order.

**Comment by mweinelt** [comment]
Thanks!

**Pull request merged**